lib/http.ex

```diff
@@ -43,6 +44,19 @@ defmodule ChromeRemoteInterface.HTTP do
     {:error, reason}
   end
 
-  defp format_body(""), do: "{}"
-  defp format_body(body), do: body
+  defp format_body(""), do: {:ok, "{}"}
+  defp format_body(body), do: {:ok, body}
+
+  defp decode(body, tries \\ 0) do
+    case Poison.decode(body) do
+      {:ok, json} -> {:ok, json}
+      {:error, _reason} when tries == 0 ->
+        encode(body) |> decode(1)
+      error -> error
+    end
+  end
+
+  defp encode(body) do
+    Poison.encode!(body)
+  end
```

Comment:
Maybe temporarily, we can just pass through the response body instead of wrapping it here.

```elixir
defp decode(body) do
  case Poison.decode(body) do
    {:ok, json} -> {:ok, json}
    {:error, _} -> {:ok, body}
  end
end
```

The ok tuple is probably a bit redundant since it would never fail, but curious to hear your thoughts.

Reply:
probably, but just done for styling

Reply:
want me to change?

Reply:
@bcardarella that’d be great!

Reply:
done

Reply:
Hmm, something weird happened, the changes didn't get pushed up into this PR

Reply:
<img width="981" alt="screen shot 2017-10-25 at 3 38 48 pm" src="https://user-images.githubusercontent.com/18524/32019265-a03afd8a-b99a-11e7-89eb-40e03a06247b.png">

Do you see the same diff as me?

Reply:
oh wait, I misunderstood the requested change